docs/configuration.md

```diff
@@ -102,7 +102,7 @@ The workbench provides a hierarchical configuration structure at `src/prompt_sir
 config/
 ├── config.yaml           # Main configuration file
 ├── attack/              # Attack configurations
-│   ├── agentdojo.yaml   # Default AgentDojo attack
+│   ├── template_string.yaml   # Default template string attack
```

Comment:
I think we could actually call this `agentdojo_template.yaml` and keep there what we had earlier. This is because this is anyways specific to this config